com.liferay.blade.api/src/com/liferay/blade/api/XMLFile.java

```diff
@@ -21,4 +21,6 @@
 public interface XMLFile extends SourceFile {
 
 	Collection<SearchResult> findElement(String elementName, String elementValue);
+
+	Collection<SearchResult> findLegacyDependency(String groupId, String artifactId);
```

Comment:
We instead could add a way to do "XPath" searches here.  Then the calling method that is looking for legacy dependencies could do an XPath search.  Can you add this?